lib/internal/Magento/Framework/Escaper.php

```diff
@@ -37,6 +37,19 @@ class Escaper
      */
     private $escapeAsUrlAttributes = ['href'];
 
+    /**
+     * Escaper constructor.
+     * @param \Magento\Framework\ZendEscaper $escaper
+     * @param \Psr\Log\LoggerInterface $logger
+     */
+    public function __construct(
+        \Magento\Framework\ZendEscaper $escaper,
+        \Psr\Log\LoggerInterface $logger
+    ) {
+        $this->escaper = $escaper;
+        $this->logger = $logger;
```

Comment:
Such change is backward-incompatible. 

Please inject new dependencies in backward-compatible manner: https://devdocs.magento.com/guides/v2.3/contributor-guide/backward-compatible-development/#adding-a-constructor-parameter

Reply:
I added null as default to keep all instances that are instantiated directly unbroken.

Reply:
@maciejslawik please remove private deprecated methods as you did before and add property initialization into constructor (as in devdoc I mentioned).